core/templates/dev/head/base.html

```diff
@@ -220,9 +216,58 @@
                             </ul>
                           </li>
                         {% endif %}
-                      </ul>
+                    {% if nav_mode != 'create' and nav_mode != 'explore' %}
+                    <div class="pull-right oppia-navbar-button-container">
+                      <!-- NOTE: If you change the links here, please make the corresponding change in the 'no results' message. -->
+                      {% if not username %}
+                        <!-- User must complete the registration process. -->
+                        <a class="btn oppia-navbar-add-exploration-button protractor-test-create-exploration oppia-transition-200"
+                           href ng-click="onRedirectToLogin('{{login_url}}')">
+                          <span class="oppia-navbar-add-exploration-button-plus">+</span>
+                          <span>Create Exploration</span>
+                        </a>
+                      {% else %}
+                        <a class="btn oppia-navbar-add-exploration-button protractor-test-create-exploration oppia-transition-200"
+                           ng-click="showCreateExplorationModal()">
+                          <span class="oppia-navbar-add-exploration-button-plus">+</span>
+                          <span>Create Exploration</span>
+                        </a>
+                        {% if allow_yaml_file_upload %}
+                          <a class="btn oppia-navbar-add-exploration-button oppia-transition-200"
+                             ng-click="showUploadExplorationModal()">
+                            <span class="oppia-navbar-add-exploration-button-plus">+</span>
+                            <span>Upload Exploration</span>
+                          </a>
+                        {% endif %}
+                      {% endif %}
                     </div>
+                    <li class="dropdown oppia-clickable-dropdown pull-right" >
+                      <a class="oppia-navbar-profile-dropdown-toggle" style="font-size: 1em; padding-top: 17px;" href="/forum">
+                    Forum
+                    </a>
+                    </li>
+                    <li class="dropdown oppia-clickable-dropdown pull-right">
+                      <a class="oppia-navbar-profile-dropdown-toggle" style="font-size: 1em; padding-top: 17px;" href="/about">
+                      About</a>
+                      <ul class="dropdown-menu oppia-navbar-profile-dropdown __web-inspector-hide-shortcut__" ng-mouseover="onMouseoverDropdownMenu($event)" ng-mouseleave="onMouseoutDropdownMenu($event)">
+                <li><a href="/about#our-mission">Mission</a></li>
```

Comment:
Indentation is a bit off. This should be indented by an additional 2 spaces relative to the ul.